src/command/game/boxScore.js

```diff
@@ -71,26 +73,49 @@ const createTeamBoxScore = team => {
 
     const totalRebounds = +rebounds_offensive + +rebounds_defensive;
 
-    boxScoreTable.push(
-      alignCenter([
-        bold(`${first_name} ${last_name}`),
-        bold(position_short),
-        checkGameHigh(players, 'minutes', minutes),
-        `${field_goals_made}-${field_goals_attempted}`,
-        `${three_pointers_made}-${three_pointers_attempted}`,
-        `${free_throws_made}-${free_throws_attempted}`,
-        checkGameHigh(players, 'plus_minus', plus_minus),
-        checkGameHigh(players, 'rebounds_offensive', rebounds_offensive),
-        checkGameHigh(players, 'rebounds_defensive', rebounds_defensive),
-        checkOverStandard(totalRebounds, 10),
-        checkGameHigh(players, 'assists', assists),
-        checkGameHigh(players, 'steals', steals),
-        checkGameHigh(players, 'blocks', blocks),
-        checkGameHigh(players, 'turnovers', turnovers),
-        checkGameHigh(players, 'fouls', fouls),
-        checkGameHigh(players, 'points', points),
-      ])
-    );
+    if (player.plus_minus === '0') {
+      boxScoreTable.push(
+        alignCenter([
+          bold(`${first_name} ${last_name}`),
+          bold(position_short),
+          checkGameHigh(players, 'minutes', minutes, 35),
+          `${field_goals_made}-${field_goals_attempted}`,
+          `${three_pointers_made}-${three_pointers_attempted}`,
+          `${free_throws_made}-${free_throws_attempted}`,
+          checkOverStandard(plus_minus, 15),
```

Comment:
This is the only difference place between two cases, right?
I don't think this is necessary, cuz it's still the highest in the team even though it's 0.

Reply:
I got it, I will remove that part of code and pull the request again. 
Btw thanks for the reply.

Reply:
no problem.